ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -23,19 +24,29 @@
 """
 
 
+def ordered_representer(dumper, data):
+    return dumper.represent_mapping('tag:yaml.org,2002:map', data.items())
+
+yaml.add_representer(OrderedDict, ordered_representer, Dumper=yaml.SafeDumper)
+
+
 def generate_catalog_yaml_metadata(path, metadata, _type='zipped-medias'):
+    package_metadata = OrderedDict()
+    for k in ('name', 'description', 'language', 'version', 'url', 'size', 'sha256sum', 'type'):
+        if k == 'version':
+            package_metadata[k] = date.today().isoformat()
+        elif k == 'sha256sum':
+            package_metadata[k] = get_file_sha256(path)
+        elif k == 'size':
+            package_metadata[k] = get_file_size(path)
+        elif k == 'type':
+            package_metadata[k] = _type
+        else:
+            package_metadata[k] = metadata[k]
```

Comment:
Huhu, I haven't think about this construction. I was thinking of `OrderedDict(name=..., description=...)` which, of course, doesn't keep the order.

I've written the `for` loop telling me it was pretty complex for nothing.